website/docs/reference/global-configs/resource-type.md

```diff
@@ -4,8 +4,18 @@ id: "resource-type"
 sidebar: "resource type"
 ---
 
+<VersionBlock lastVersion="1.8">
+
 The `--resource-type` and `--exclude-resource-type` flags include or exclude resource types from the `dbt build`, `dbt clone`, and `dbt list` commands.
 
+</VersionBlock>
+
+<VersionBlock firstVersion="1.9">
+
+The `--resource-type` and `--exclude-resource-type` flags include or exclude resource types from the `dbt build`, `dbt test`, `dbt clone`, and `dbt list` commands.
```

Comment:
I believe this resolves the comment in https://github.com/dbt-labs/docs.getdbt.com/pull/6126#discussion_r1781351928.

Namely, as-of v1.9, there are 4 different subcommands that support these flags, and this sentence lists all of them: `build`, `test`, `clone`, and `list`.